pyacq/core/rpc.py

```diff
@@ -240,7 +268,23 @@ def __init__(self, client, method):
         self.method = method
         
     def __call__(self, *args, **kwds):
-        return self.client._call_method(self.method, *args, **kwds)
+        # This little hack could be a help for manger remote method when dealing with node
+        # for instance:
+        # nodename = mynodeproxy.name
+        # manager.create_node_outputs(nodename, streamdef)
+        # could be directly be
+        # manager.create_node_outputs(mynodeproxy, streamdef)
```

Comment:
OK.
I remove this for the moment.
And we will think about a global solution in a separate PR.